src/ViewItem.js

```diff
@@ -216,6 +219,37 @@ class ViewItem extends React.Component {
     this.props.mutator.query.update({ layer: 'copyItem' });
   }
 
+  getActionMenu({ onToggle }, item) {
```

Comment:
Same oddity here with `item`, but should be another scope of work.

Reply:
I'll move it into the `getActionMenu`-method instead.